Libraries/LibCrypto/Curves/Ed448.h

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) 2024, Altomani Gianluca <altomanigianluca@gmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <AK/ByteBuffer.h>
+
+namespace Crypto::Curves {
+
+class Ed448 {
+public:
+    size_t key_size() { return 57; }
+    size_t signature_size() { return 114; }
```

Comment:
these member functions should at least be const. 

```
size_t key_size() const { return 57; }
```
if not `constexpr` as well

```
constexpr size_t key_size() const { return 57; }
```